velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -686,12 +731,29 @@ BufferPtr VectorFuzzer::fuzzNulls(vector_size_t size) {
 
 variant VectorFuzzer::randVariant(const TypePtr& arg) {
   VELOX_CHECK(arg->isPrimitiveType());
+  if (isDecimalKind(arg->kind())) {
```

Comment:
I'm removing randVariant method in #3834

Reply:
Ack.